src/app/utils/db.js

```diff
@@ -0,0 +1,15 @@
+import Dexie from 'dexie';
```

Comment:
```suggestion
"use client"

import Dexie from 'dexie';
```
Well, indexdb is not existed in nodejs, right?